packages/terminal-extension/src/index.ts

```diff
@@ -113,7 +113,9 @@ function activate(app: JupyterLab, services: IServiceManager, mainMenu: IMainMen
     CommandIDs.toggleTheme
   ].forEach(command => {
     palette.addItem({ command, category });
-    menu.addItem({ command });
+    if (command !== CommandIDs.createNew) {
+      menu.addItem({ command });
+    }
```

Comment:
No other activities have a "New" item in their menu. The idea is to consolidate all "new" things into a unified launcher experience.